.github/workflows/publish-too-many-hooks.yml

```diff
@@ -40,17 +43,22 @@ jobs:
         run: bun run build
         working-directory: packages/too-many-hooks
 
+      - name: Swap READMEs
+        run: |
+          mv ./packages/too-many-hooks/README.md ./packages/too-many-hooks/dev.README.md
+          mv ./README.md ./packages/too-many-hooks/README.md
```

Comment:
Do we need to keep the initial dev README around for publication? If not, this could just be a move command that replaces the existing file.